.circleci/src/jobs/push-docker-image.yml

```diff
@@ -15,7 +15,6 @@ steps:
   - run:
       name: Docker login
       command: echo "$DOCKERHUB_PASS" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
-  - run: . ~/.profile; audius-compose build --prod "<< parameters.service >>"
```

Comment:
assuming this is safe to remove given that the push command already runs this same build command? there was some discussion about this before, so i want to double-check